src/SurveyHandler.ts

```diff
@@ -794,6 +795,10 @@ class SurveyHandler {
               startRangeLabel = "Low Effort";
               endRangeLabel = "High Effort";
             }
+            else {
```

Comment:
check if the question has question.anchorMetricName attribute set before accessing the custom Metrics